packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -52,15 +76,17 @@ const getEnterCode = (views, { isGeneralLogin = true }) => {
 			delete req.session?.enterCode?.newCode;
 		}
 
+		const sqlUser = await isFeatureActive(FLAG.ENROL_USERS);
```

Comment:
I think making it clear this is a feature flag in the variable name will help with clarity in the rest of the code.

Reply:
agreed yes, I did this before I started on the split of the feature flag, but this was a poor choice